qcfractal/interface/collections/collection.py

```diff
@@ -33,7 +30,7 @@ def __init__(self, name, **kwargs):
         """
 
         self.client = kwargs.pop("client", None)
-        if (self.client is not None) and (not isinstance(self.client, FractalClient)):
+        if (self.client is not None) and not (self.client.__class__.__name__ == "FractalClient"):
```

Comment:
Is this safe? Wouldn't the `isinstance` be a much safer call than any class named "FractalClient"? This would also make it so subclasses of FractalClient (should remote users chose to make one) won't work here.

Reply:
Not particularly safe, but would cause circular imports otherwise.

Reply:
Ah. Those. Yes, lets avoid those first